source/declarations.tex

```diff
@@ -5386,8 +5389,8 @@
 its type \tcode{T4} is adjusted to type ``\cvqual{cv1} \tcode{T4}''\iref{conv.qual}
 and the temporary materialization conversion\iref{conv.rval} is applied.
 In any case,
-the reference is bound to the resulting glvalue
-(or to an appropriate base class subobject).
+the reference is bound to the object or function to which the resulting glvalue
+refers (or to an appropriate base class subobject).
```

Comment:
How about just:

is bound to the resulting glvalue -> is bound to the glvalue result

Reply:
Sounds good